### PR TITLE
Consolidate helloworld sample apps output and make sure instruction consistent with source code

### PR DESCRIPTION
**Pull request by chaodaiG**
Consolidating helloworld sample apps output, so that they are using the same pattern, also makes sure README.md files are consistent with source code

### CODE REVIEW

**Comment by chaodaiG** [comment]
/cc @adrcunha 

Reference to what we discussed in PR https://github.com/knative/docs/pull/1083

File: docs/serving/samples/hello-world/helloworld-go/README.md [hunk 1]
```diff
@@ -60,13 +60,13 @@ recreate the source files from this folder.
    # Use the offical Golang image to create a build artifact.
    # This is based on Debian and sets the GOPATH to /go.
    # https://hub.docker.com/_/golang
-   FROM golang as builder
+   FROM golang:1.12 as builder
```
Comment (adrcunha) [inline docs/serving/samples/hello-world/helloworld-go/README.md]: Why 1.12 explicitly?

File: docs/serving/samples/hello-world/helloworld-go/README.md [hunk 2]
```diff
@@ -60,13 +60,13 @@ recreate the source files from this folder.
...
 
    # Copy local code to the container image.
    WORKDIR /go/src/github.com/knative/docs/helloworld
    COPY . .
 
-   # Build the helloworld command inside the container.
```
Comment (adrcunha) [inline docs/serving/samples/hello-world/helloworld-go/README.md]: That's actually a "command". I would write "Build the helloworld binary inside the container" though.

File: docs/serving/samples/hello-world/helloworld-kotlin/README.md [hunk 3]
```diff
@@ -109,7 +109,7 @@ recreate the source files from this folder.
    ```docker
    # Use the official gradle image to create a build artifact.
    # https://hub.docker.com/_/gradle
-   FROM gradle as builder
+   FROM gradle:4.10 as builder
```
Comment (adrcunha) [inline docs/serving/samples/hello-world/helloworld-kotlin/README.md]: Same question about a specific version of gradle here.

File: docs/serving/samples/hello-world/helloworld-nodejs/README.md [hunk 4]
```diff
@@ -93,7 +93,7 @@ recreate the source files from this folder.
    # Copy application dependency manifests to the container image.
    # A wildcard is used to ensure both package.json AND package-lock.json are copied.
    # Copying this separately prevents re-running npm install on every code change.
-   COPY package*.json ./
+   COPY package.json package*.json ./
```
Comment (adrcunha) [inline docs/serving/samples/hello-world/helloworld-nodejs/README.md]: Really? COPY doesn't recognized package.json when using the wildcard? If so, please update the comment.

File: docs/serving/samples/hello-world/helloworld-go/README.md (hunk 1 above)
Comment (chaodaiG) [reply docs/serving/samples/hello-world/helloworld-go/README.md]: The purpose of this PR is consolidating README files, all these changes are due to inconsistency between source file and README. Preferably I would pick the one comes later, but it's not doable as they all lost history in github UI due to recent movement. So I just blindly picked one that I thought being later

Comment (chaodaiG) [reply docs/serving/samples/hello-world/helloworld-go/README.md]: for example this line comes from this file:
https://github.com/knative/docs/blob/52837ad26ad4cd2e3e93a5c17e8a32941d1e6a23/docs/serving/samples/hello-world/helloworld-go/Dockerfile#L4

Comment (adrcunha) [reply docs/serving/samples/hello-world/helloworld-go/README.md]: Thanks for clarifying.

File: docs/serving/samples/hello-world/helloworld-go/README.md (hunk 2 above)
Comment (chaodaiG) [reply docs/serving/samples/hello-world/helloworld-go/README.md]: same here

File: docs/serving/samples/hello-world/helloworld-kotlin/README.md (hunk 3 above)
Comment (chaodaiG) [reply docs/serving/samples/hello-world/helloworld-kotlin/README.md]: same here

File: docs/serving/samples/hello-world/helloworld-nodejs/README.md (hunk 4 above)
Comment (chaodaiG) [reply docs/serving/samples/hello-world/helloworld-nodejs/README.md]: Nice catch. I just tested out, wildcard works for package.json. Updated

**Pull request merged**